config/database.php

```diff
@@ -1,126 +1,30 @@
-            'host'      => env('DB_HOST', '192.168.0.13'),
-            'database'  => env('DB_DATABASE', 'raft'),
-            'username'  => env('DB_USERNAME', 'root'),
-            'password'  => env('DB_PASSWORD', 'isnear'),
-            'charset'   => 'utf8',
-            'collation' => 'utf8_unicode_ci',
-            'prefix'    => '',
-            'strict'    => false,
-        ],
-
-        'pgsql' => [
-            'driver'   => 'pgsql',
-            'host'     => env('DB_HOST', 'localhost'),
-            'database' => env('DB_DATABASE', 'forge'),
-            'username' => env('DB_USERNAME', 'forge'),
-            'password' => env('DB_PASSWORD', ''),
-            'charset'  => 'utf8',
-            'prefix'   => '',
-            'schema'   => 'public',
-        ],
-
-        'sqlsrv' => [
-            'driver'   => 'sqlsrv',
-            'host'     => env('DB_HOST', 'localhost'),
-            'database' => env('DB_DATABASE', 'forge'),
-            'username' => env('DB_USERNAME', 'forge'),
-            'password' => env('DB_PASSWORD', ''),
-            'charset'  => 'utf8',
-            'prefix'   => '',
-        ],
-
-    ],
-
-    /*
-    |--------------------------------------------------------------------------
-    | Migration Repository Table
-    |--------------------------------------------------------------------------
-    |
-    | This table keeps track of all the migrations that have already run for
-    | your application. Using this information, we can determine which of
-    | the migrations on disk haven't actually been run in the database.
-    |
-    */
-
-    'migrations' => 'migrations',
-
-    /*
-    |--------------------------------------------------------------------------
-    | Redis Databases
-    |--------------------------------------------------------------------------
-    |
-    | Redis is an open source, fast, and advanced key-value store that also
-    | provides a richer set of commands than a typical key-value systems
-    | such as APC or Memcached. Laravel makes it easy to dig right in.
-    |
-    */
-
-    'redis' => [
-
-        'cluster' => false,
-
-        'default' => [
-            'host'     => '127.0.0.1',
-            'port'     => 6379,
-            'database' => 0,
-        ],
-
-    ],
-
+  'fetch' => PDO::FETCH_CLASS,
+  'default' => env('DB_CONNECTION', 'mysql'),
+  'connections' => [
+    'mysql' => [
+      'driver'    => 'mysql',
+      'host'      => env('DB_HOST', '192.168.0.13'),
+      'database'  => env('DB_DATABASE', 'raft'),
+      'username'  => env('DB_USERNAME', 'root'),
+      'password'  => env('DB_PASSWORD', 'isnear'),
+      'charset'   => 'utf8',
+      'collation' => 'utf8_unicode_ci',
+      'prefix'    => '',
+      'strict'    => false,
+    ]
+  ],
+  'migrations' => 'migrations',
+  'redis' => [
+    'cluster' => false,
+    'default' => [
+        'host'     => '127.0.0.1',
+        'port'     => 6379,
+        'database' => 0,
+    ]
+  ]
 ];
+
+?>
```

Comment:
I'm removing all this because it's just in the way. If we ever want to see how to configure the other connection types we can just look at the laravel repo. the goal is to _only have code checked in that is used by the application_.